build/tradable.js

```diff
@@ -0,0 +1,26 @@
+const builtUntradable = ['Warframe', 'Throwing', 'Shotgun', 'Rifle', 'Pistol', 'Melee', 'Sword And Shield']
+const tradableConditions = (i) => !(builtUntradable.includes(i.type) && i.name.match(/Prime/ig))
+
+const tradableTypes = ['Upgrades', 'Arcane', 'Fish', 'Focus Lens', 'Relic', 'Secondary Mod', 'Shotgun Mod',
+  'Warframe Mod', 'Companion Mod', 'Archwing Mod', 'K-Drive Mod', 'Melee Mod', 'Arch-Melee Mod', 'Necramech Mod',
+  'Cut Gem', 'Captura', 'Primary Mod', 'Sentinel Mod', 'Kubrow Mod', 'Kavat Mod']
+const untradableTypes = ['Skin', 'Medallion', 'Key', 'Extractor', 'Pets', 'Ship Decoration',
+  'Glyph', 'Sigil', 'Fur Color', 'Syandana', 'Fur Pattern', 'Color Palette', 'Node', 'Exalted Weapon']
+const tradableRegex = /(Prime|Vandal|Wraith\w|\wWraith|Rakta|Synoid|Sancti|Vaykor|Telos|Secura|Ayatan|Prisma|DamagedMech)(?!Derelict)/i
+const untradableRegex = /(Glyph|Mandachord|Greater.*Lens|Sugatra|\[|SentinelWeapons|Toroid|Bait|([A-Za-z]+ (Relic)))|Umbral|Sacrificial/i
+
+/**
+ * Check if an item is tradable
+ * @param {module:warframe-items.Item} item
+ * @returns {boolean}
+ */
+module.exports = (item) => {
+  const notFiltered = !untradableTypes.includes(item.type) &&
+    !item.name.match(untradableRegex) &&
+    !item.uniqueName.match(untradableRegex) &&
+    (item.productCategory ? !item.productCategory.match(/(SpecialItems)/) : true)
+  const tradableByType = !!(tradableTypes.includes(item.type) && notFiltered)
+  const tradableByName = !!((item.uniqueName.match(tradableRegex) || item.name.match(tradableRegex)) && notFiltered)
+  const tradableByProp = !!((item.isAugment) && notFiltered)
```

Comment:
Is having `!!` with additional `()` enclosing the booleans meant for readability? I personally don't like it much, because it's rare to see it.

Reply:
it's for massaging values to booleans, since `notFiltered` can be an object, and could ultimately trickle into being the value on `tradable`, which i also want to for sure be a boolean.

Reply:
That's what i thought it's not needed to do that but i think it's good practice, so what do you think of reducing the amount of parenthesis? Like:
```suggestion
  const tradableByType = !!(tradableTypes.includes(item.type) && notFiltered)
  const tradableByName = !!(item.uniqueName.match(tradableRegex) || item.name.match(tradableRegex) && notFiltered)
  const tradableByProp = !!(item.isAugment && notFiltered)
```

Reply:
added